src/Stratis.StratisD.Test/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Protocol;
+using Stratis.Bitcoin;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Features.Api;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Utilities;
+using Stratis.StratisD.Test.Networks;
+
+namespace Stratis.StratisD.Test
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            try
+            {
+                Network network = null;
+                if (args.Contains("-network=StratisPosTestNetwork"))
+                    network = new StratisPosTestNetwork();
```

Comment:
probably its not correct to call it Stratis maybe just `PosTestNetwork` and `PowTestNetwork`?

Reply:
Or maybe even add the word Generated `GeneratedPowTestNetwork`